docker/Dockerfile.pyt_build

```diff
@@ -0,0 +1,71 @@
+ARG BASE_IMAGE=pytorch/pytorch:2.6.0-cuda12.6-cudnn9-devel
+ARG DEVEL_IMAGE=devel
+
+FROM ${BASE_IMAGE} AS devel
+WORKDIR /workspace/deps
+
+RUN ls -la /usr/local/cuda && echo "/usr/local/cuda exists" || echo "/usr/local/cuda does NOT exist"
+
+# Set CUDA_HOME and update PATH and LD_LIBRARY_PATH
+ENV CUDA_HOME=/usr/local/cuda-12.6
+ENV LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${CUDA_HOME}/compat:${LD_LIBRARY_PATH}
+
+# Install git first
+RUN apt-get update && \
+    apt-get install -y \
+    git \
+    && apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Also create symbolic link for libcuda.so.1 as a fallback
+RUN mkdir -p /usr/lib/x86_64-linux-gnu/ && \
+    ln -s ${CUDA_HOME}/compat/libcuda.so.1 /usr/lib/x86_64-linux-gnu/libcuda.so.1
+
+RUN git clone -b core_r0.9.0 https://github.com/NVIDIA/Megatron-LM.git megatron-lm && \
+    pip install -e ./megatron-lm
+
+RUN pip install torchx gin-config torchmetrics==1.0.3 typing-extensions iopath
+
+RUN pip install --no-cache setuptools==69.5.1 setuptools-git-versioning scikit-build && \
+  git clone --recursive -b main https://github.com/pytorch/FBGEMM.git fbgemm && \
```

Comment:
RUN git clone --recursive -b main \
      https://github.com/pytorch/FBGEMM.git /tmp/fbgemm && \
    cd /tmp/fbgemm/fbgemm_gpu && \
    
    put to tmp?
and later     rm -rf /tmp/fbgemm